### PR TITLE
Blog post: monitor mode

### PR DESCRIPTION
**Pull request by ereslibre**
Fixes: https://github.com/kubewarden/kubewarden.io/issues/92

### CODE REVIEW

File: content/blog/2022/05/monitor-mode.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+With the Monitor mode, we are able to deploy new policies so that they
+will not reject or mutate any request they might be targeting. The
+rest of the mechanism for policy evaluation will work as usual, so
+that you can still:
+
+- Inspect the `policy-server` logs and traces for actions this policy
+  would have taken had it been in `protect` mode.
+
+- Inspect the `policy-server` metrics, given the mode is now part of
+  the metric baggage.
```
Comment (ereslibre) [inline content/blog/2022/05/monitor-mode.md:46]: I was seeing some shanenigans here rendering locally with `hugo`, but it seems it's also an issue in the published CSS now: https://www.kubewarden.io/blog/2022/02/multiplatform-kubewarden/

Comment (ereslibre) [reply content/blog/2022/05/monitor-mode.md:46]: /cc @jordojordo 🙏

Comment (jordojordo) [reply content/blog/2022/05/monitor-mode.md:46]: There are some rules from the project theme sass that is causing some trouble there, created an [issue](https://github.com/kubewarden/kubewarden.io/issues/95) to track this.

Comment (flavio) [reply content/blog/2022/05/monitor-mode.md:46]: the issue is now fixed, thanks @jordojordo!

**Review by flavio — APPROVED** [review]

**Pull request merged**